frontend/src/components/Item/CollectionTabs.vue

```diff
@@ -40,11 +43,14 @@ const children = computed(() => {
 watch(
   () => props.item,
   async (item) => {
-    if (!children.value) {
-      loading.value = true;
-      await items.fetchAndAddCollection(item.Id);
-      loading.value = false;
+    // only try to load children if there aren't any children items in the itemStore yet
```

Comment:
```suggestion
    /**
    * Only try to load children if there aren't any children items in the itemStore yet
    */
```

We tend to prefer this syntax :)

Reply:
I'll update this for consistency, but I'm curious why, I'll put some thoughts in Matrix.